internal/controllers/suite_test.go

```diff
@@ -17,31 +17,78 @@ limitations under the License.
 package controllers_test
 
 import (
-	"crypto/x509"
+	"context"
+	"io/fs"
 	"log"
+	"net/http"
 	"os"
 	"path/filepath"
 	"testing"
 
+	"github.com/stretchr/testify/mock"
 	"github.com/stretchr/testify/require"
 	"k8s.io/apimachinery/pkg/api/meta"
 	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
 	"k8s.io/client-go/rest"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/envtest"
-	"sigs.k8s.io/controller-runtime/pkg/manager"
 
 	helmclient "github.com/operator-framework/helm-operator-plugins/pkg/client"
+	"github.com/operator-framework/rukpak/api/v1alpha2"
 	"github.com/operator-framework/rukpak/pkg/source"
-	"github.com/operator-framework/rukpak/pkg/util"
+	"github.com/operator-framework/rukpak/pkg/storage"
 
 	"github.com/operator-framework/operator-controller/internal/controllers"
 	"github.com/operator-framework/operator-controller/pkg/scheme"
 	testutil "github.com/operator-framework/operator-controller/test/util"
 )
 
+// MockUnpacker is a mock of Unpacker interface
```

Comment:
Nit: move the mocks to a separate package and/or file to make them easier to find?

Reply:
Added as #895.